docs/FAQ/FAQ-Synchronization.md

```diff
@@ -185,13 +185,13 @@ tmpfs           2.0G     0  2.0G   0% /sys/fs/cgroup
 /dev/sdb1       7.8G   18M  7.4G   1% /mnt
 ```
 
-[Choose the docker fragment](https://github.com/btcpayserver/btcpayserver-docker#generated-docker-compose-) for the amount of storage you aim to keep.
+[Choose the docker fragment](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#generated-docker-compose) for the amount of storage you aim to keep.
 
-Then [prune your node](https://github.com/btcpayserver/btcpayserver-docker#how-i-can-prune-my-nodes).
+Then [prune your node](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#how-i-can-prune-my-node-s).
```

Comment:
```suggestion
Then [prune your node](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#how-i-can-prune-my-nodes).
```

Reply:
Thanks for pointing that out. Actually this is a bit more complicated: The Vuepress slug function seems to diverge from what GitHub's markdown parser generates … will have to dig a bit deeper and see how we can make it work on GH as well as on the doc site.

Reply:
Hope I fully addressed it with 60a5ccb.